test/support/connection_test_helper.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Janus.ConnectionTest.Helper do
```

Comment:
This name doesn't resonate with me. It contains helpers only for testing callbacks. I would scope group those modules together.